src/main/webapp/app/admin/user-management/user-management.component.ts

```diff
@@ -419,6 +423,58 @@ export class UserManagementComponent implements OnInit, OnDestroy {
         });
     }
 
+    /**
+     * Selects/Unselects all (filtered) users.
+     */
+    toggleAllUserSelection() {
+        const usersWithoutCurrentUser = this.usersWithoutCurrentUser;
+        if (this.selectedUsers.length === usersWithoutCurrentUser.length) {
+            // Clear all users
+            this.selectedUsers = [];
+        } else {
+            // Add all users
+            this.selectedUsers = [...usersWithoutCurrentUser];
+        }
+    }
+
+    /**
+     * Gets the users without the current user.
+     */
+    get usersWithoutCurrentUser() {
+        return this.users.filter((user) => this.currentAccount && this.currentAccount.login !== user.login);
+    }
+
+    /**
+     * Selects/Unselects a user.
+     */
+    toggleUser(user: User) {
+        const index = this.selectedUsers.indexOf(user);
+        if (index > -1) {
+            this.selectedUsers.splice(index, 1);
+        } else {
+            // Add all users
+            this.selectedUsers.push(user);
+        }
+    }
+
+    /**
+     * Delete all selected users.
+     */
+    deleteAllSelectedUsers() {
+        const logins = this.selectedUsers.map((user) => user.login!);
+        this.userService.deleteUsers(logins).subscribe({
+            next: () => {
+                this.eventManager.broadcast({
+                    name: 'userListModification',
```

Comment:
You return the number of users. Either use them in the broadcast or remove it from the response.

Reply:
I use it in the broadcast. This is done implicit since the params are part of the header.